src/api/todos.ts

```diff
@@ -5,4 +5,10 @@ export const getTodos = (userId: number) => {
   return client.get<Todo[]>(`/todos?userId=${userId}`);
 };
 
-// Add more methods here
+export const addTodo = (title: string, userId: number, completed = false) => {
+  return client.post<Todo>('/todos', { title, userId, completed });
+};
```

Comment:
I think, this is more clear for understand
```suggestion
export const addTodo = (newTodo: Omit<Todo, 'id'>) => {
  return client.post<Todo>('/todos', newTodo);
};
```

Reply:
Thank you for your comments!
 I would like to leave the last one as it is)

Reply:
> I think, this is more clear for understand

It's a good comment, let's implement it.

Reply:
It's a good comment, let's implement it